engines/bops_config/app/controllers/bops_config/local_authorities_controller.rb

```diff
@@ -2,18 +2,18 @@
 
 module BopsConfig
   class LocalAuthoritiesController < ApplicationController
-    before_action :set_local_authorities, only: %i[index]
```

Comment:
was there any issue and you had to remove that?

Reply:
[I think I was the issue 😄](https://github.com/unboxed/bops/pull/2157#discussion_r1952490086)